thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -202,6 +201,8 @@ public Builder setInstallationID(
          *
          * @param app Kii Cloud Application.
          * @param owner Specify who uses the ThingIFAPI.
+         * @param actionTypes the format of key of actionTypes is "[alias]:[actionName]".
```

Comment:
If alias name is "AirConditionerAlias" and action name is `turnPower`, Which is correct in following 2 strings?

1. `AirConditionerAlias:turnPower`
1. `[AirConditionerAlias]:[turnPower]`